lib/Command.js

```diff
@@ -0,0 +1,88 @@
+const { Utils } = require('../src/utils.js');
+const utils = new Utils;
```

Comment:
you are instantiating a new utils instance for every command.
I would change that to either:
- Export new Utils() which will make it an object available from everywhere
- Make Utils a fully static class and never instantiate it ||or|| export a module being a group of function instead of a class
- Make Utils a property of Assyst, and `get Utils() { return this.assyst.utils }`